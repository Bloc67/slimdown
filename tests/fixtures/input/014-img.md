![alt text](http://example.com/foo_underscore_test.jpg)

![](http://example.com/foo_underscore_test.jpg)
