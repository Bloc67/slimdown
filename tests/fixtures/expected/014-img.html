<p><img src='http://example.com/foo_underscore_test.jpg' alt='alt text'></p>

<p><img src='http://example.com/foo_underscore_test.jpg' alt=''></p>